plugins/flatpak/gs-flatpak.c

```diff
@@ -2808,24 +2811,24 @@ gs_flatpak_get_list_for_install (GsFlatpak *self, GsApp *app,
 		/* already installed? */
 		app_tmp = gs_flatpak_create_app (self, FLATPAK_REF (xref_related));
 		ref_display = gs_flatpak_app_get_ref_display (app_tmp);
-		if (g_hash_table_contains (hash_installed, ref_display)) {
+		if (!is_update && g_hash_table_contains (hash_installed, ref_display)) {
 			g_debug ("not adding related %s as already installed", ref_display);
```

Comment:
Could you add a `continue` after this line and then un-indent the code inside the `else { }` branch one level to the left? I think the code would be clearer that way.

Reply:
Good idea.